int32 distance
int32[] vector_tag
int32[] vector_plane
int32[] T
# distance 代表一个距离尺度（待定）
# vector分别代表标定后地面和tag的法向量